src/app/merge-duplicates/merge-duplicates-table/merge-duplicates-table.component.ts

```diff
@@ -0,0 +1,156 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { DatepickerHeaderTodayComponent } from '../../shared/dynamic-form/components/datepicker-header-today/datepicker-header-today.component';
+import { DEFAULT_DATE_FORMAT } from '../../_constants/common';
+import { MergeDuplicateDto } from '../../_models/mergeDuplicateDto';
+import { MergeDuplicateService } from '../../_services/api/mergeDuplicate.service';
+import { MergeDuplicateContactService } from '../../_services/api/mergeDuplicateContact.service';
+
+@Component({
+  selector: 'app-merge-duplicates-table',
+  templateUrl: './merge-duplicates-table.component.html',
+  styleUrls: ['./merge-duplicates-table.component.scss'],
+})
+export class MergeDuplicatesTableComponent implements OnInit, OnDestroy {
+  header = DatepickerHeaderTodayComponent;
+  defaultDateFormat = DEFAULT_DATE_FORMAT;
+
+  private subscriptions: Subscription[] = [];
+
+  mergeDuplicates: MergeDuplicateDto[];
+
+  @Input() type: string;
+
+  displayedColumns: string[];
+  hideChildren: number[] = [];
+
+  size = 10;
+  offset = 0;
+
+  constructor(
+    private mergeDuplicatesService: MergeDuplicateService,
+    private mergeDuplicatesContactService: MergeDuplicateContactService
+  ) {}
+
+  ngOnInit(): void {
+    this.displayedColumns = [
+      'uuid',
+      'disease',
+      'caseClassification',
+      'personFirstName',
+      'personLastName',
+      'sex',
+      'districtUuid',
+      'healthFacilityName',
+      'reportDate',
+      'creationDate',
+      'completeness',
+      'merge',
+      'pick',
+      'hide',
+    ];
+
+    this.getMergeDuplicates();
+  }
+
+  getMergeDuplicates(concat: boolean = false): void {
+    let service = this.mergeDuplicatesService;
+    if (this.type === 'contacts') {
+      service = this.mergeDuplicatesContactService;
+    }
+    this.subscriptions.push(
+      service.getAll({ offset: this.offset, size: this.size }, false, null, true).subscribe({
```

Comment:
I think it should be null for the sort options if there are none, instead of false; otherwise it would try to create some